File: Golang/1-50/10.go
package main

import (
	"fmt"
)

func main() {
	var A, B, C, N, d int
	fmt.Scan(&N)
	fmt.Scan(&A, &B, &C)
	d = A + B + C
	if N <= d {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

File: C/1-50/10.c
#include <stdio.h>
int main() {
    long long A, B, C, N, d;
    scanf("%lld", &N);
    scanf("%lld %lld %lld", &A, &B, &C);
    d = A + B + C;
    if (N <= d) {
        printf("Yes");
    } else {
        printf("No");
    }
    return 0;
}
